pyproject.toml

```
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "prediction-market-agent-tooling"
-version = "0.9.3"
+version = "0.10.0"
```

```suggestion
version = "0.11.0"
```

0.10.0 is already out, but I wonder why github doesn't complain about confclits with main 🤔 

https://github.com/gnosis/prediction-market-agent-tooling/blob/main/pyproject.toml#L3